Maestro/subscriptions.json

```diff
@@ -1144,14 +1142,11 @@
         "vsoSourceBranch": "nightly"
       }
     },
-    // Automate opening PRs to merge aspnet release/2.1 branches back to release/3.0
+    // Automate opening PRs to merge aspnet release/2.1 branches back to release/3.0 or release/3.1.
```

Comment:
@vijayrkn @jodavis I'm leaving changes related to aspnet/BrowserLink and aspnet/Scaffolding to you